maturity-level-one/src/AutomapperConfig.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Codit.LevelOne
+{
+    public static class AutoMapperConfig
```

Comment:
Why does it need to be like this? Isn't it up to the user to correctly initialize it?

Reply:
Because otherwise the integration tests won't work (Automapper). A more elegant implementation can be done